recipe/meta.yaml

```diff
@@ -37,9 +39,17 @@ test:
 about:
   home: https://github.com/microsoft/onnxconverter-common
   summary: Common utilities for ONNX converters
+  description: |
+    The onnxconverter-common package provides common functions and utilities for use in converters from various AI frameworks to ONNX. It also enables the different converters to work together to convert a model from mixed frameworks, like a scikit-learn pipeline embedding a xgboost model.
   license: MIT
   license_file: LICENSE
+  license_family: MIT
+  dev_url: https://github.com/microsoft/onnxconverter-common
+  doc_url: https://github.com/microsoft/onnxconverter-common/blob/master/README.md
 
 extra:
   recipe-maintainers:
     - xhochy
+  skip-lints:
```

Comment:
Hmm, not sure if this should be here, because the linter is actually erroneous at the moment and needs to be fixed - I'd rather just ignore the linter message and leave this out

Reply:
Personal preference is to not get scared by the ❌ every time. But as you've noted, the linter is already failing. I've remove this. Thanks!